features/282-next-repo-re-organization/plan.md

```diff
@@ -0,0 +1,63 @@
+# Re-organize Github Repo to focus on NEXT
+
+## Repository layout
+
+The top level is the git repository.
+
+- skygear-server
+    - gateway
+    - auth gear
+    - CF gateway and related
```

Comment:
change it to cloud functions (or the name of the module?)

Reply:
maybe just called it platform? (so it is Cloud Functions and Microservices Platform, Skygear Platform)